book/website/collaboration/ppie-events/ppie-events-socials.md

```diff
@@ -0,0 +1,121 @@
+
+Consider deciding on and including a title, goal, and suggested question prompts in the session description. 
+This can provide a shared understanding among attendees of what is expected during any social session at your event. 
+
+Although most social events are traditionally unstructured to allow for emergent discussion and knowledge sharing, it can be beneficial to provide guidance or prompts to start any social sessions and then allow for unstructured conversation time later in the session.
+
+(cl-ppie-events-networking-expectations)=
+### Setting Expectations
+In addition to information about the format that your social or networking event will have, consider including clear and explicit instructions about what engagement would be suitable and appropriate for your social activities.
+
+This may be the first research-focused event for many of your attendees, and it can be helpful to surface many of the implicit social norms and practices of research events.
+
+For example, poster sessions at research events are often considered to involve short interactions between a researcher and a group of people. 
+At a poster session, an early career researcher may have the opportunity to share and explain their research project. 
+These sessions would not be the best forum for public discourse about an attendee's lived experience or for criticizing the purpose of the work that is being presented.
+Instead, poster sessions may be useful to make introductions with other attendees who are interested in a particular topic, and are an opportunity to learn about the research that has been conducted by asking questions about the research that person is presenting on their poster.
+
+Depending on the audience, your guide may need to be more or less detailed.
+By providing guidance on the type of sessions, and what is expected in each setting, you can help facilitate more constructive and collaborative interactions between attendees, and reduce the likelihood of misunderstandings.
+
+(cl-ppie-events-conversation-prompts)=
+### Sparking conversation between attendees 
+Social skills are fundamental in building relationships, sharing ideas, and collaborating on research endeavours.
+
+Providing guidance on how to start conversations and socialize effectively can help support attendees with lived experience in developing professional connections and meaningfully engage in your event.
+
+Depending on the size and timings of your event, you can consider setting prompts such as "find a fellow attendee who has the same favourite movie as you", or "introduce yourself to 5 people you have never met before, and find 1 thing you have in common". 
+Prompts such as these can provide attendees with a goal for the session and help attendees to start conversations with one another. 
+
+Good conversation prompts or ice breaker questions are both interesting for someone to think about and answer themselves, and also make attendees curious to hear everyone else’s answer.
```

Comment:
I've also seen "stickers" or "name tags" that people can wear that prompt conversation... Like, "ask me about..."!